src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ArgCoercion_V1Compat.txt

```diff
@@ -2,4 +2,10 @@
 #SETUP: ConsistentOneColumnTableResult,TableSyntaxDoesntWrapRecords
 
 >> Last([ {a:2}, {a:true, b:true, c:20}, {a:2, b:2, c:true} ]).c
-1
\ No newline at end of file
+1
+
+>> Len(GUID())
+36
+
+>> GUID() & GUID() <> ""
```

Comment:
A more interesting case would be `Len(GUID() & GUID())` ===> 72

Reply:
Added.